sdk/python/kfp/dsl/pipeline_task.py

```diff
@@ -631,6 +631,26 @@ def set_env_variable(self, name: str, value: str) -> 'PipelineTask':
             self.container_spec.env = {name: value}
         return self
 
+    @block_if_final()
+    def set_container_image(
+            self,
+            name: Union[str,
+                        pipeline_channel.PipelineChannel]) -> 'PipelineTask':
+        """Sets container  type to use when executing this task. Takes
+        precedence over @component(base_image=...)
+
+        Args:
+            name: The name of the image, e.g. "python:3.9-alpine".
```

Comment:
Given that some task methods can pipeline parameters  as arguments  and others not, what do you think about being explicit about it in the function documentation text? 

```suggestion
            name: name of the image (e.g. "python:3.9-alpine") or pipeline parameter
```